### DISCUSSION TITLE
Twitter Space w/ Viktor Farcic, Kunal Kushwaha, Hila Fish and Luca Galante on December 7!

### DISCUSSION THREAD

**Discussion by gguizza** [score-spec/spec#32] [Announcements]
On Wednesday, December 7th, our project will be featured in a [Twitter Space](https://twitter.com/i/spaces/1lPJqBDQwolxb?s=20) with Platform Engineering’s Community Manager Luca Galante (@luca_cloud), Score’s Product Manager Susa Tuenker (@_sujaya), Civo Cloud’s DevRel Manager Kunal Kushwaha (@kunalstwt), Wix’s Senior DevOps Engineer Hila Fish (@Hilafish1), and Upbound’s Developer Advocate Viktor Farcic (vfarcic). 

Join “[One YAML to rule them all](https://twitter.com/i/spaces/1lPJqBDQwolxb?s=20)” to learn more about workload specifications, cognitive load and Score. 

We can’t wait to keep on building our journey!

**Comment by sujaya-sys** [comment]
In case you missed it, the recording is available here: https://twitter.com/i/spaces/1lPJqBDQwolxb?s=20
